src/prototypes/single-article/example/example.twig

```diff
@@ -0,0 +1,327 @@
+
+              {% include '@cloudfour/components/heading/heading.twig' with {
+                level: 2,
+                content: 'Yellow Submarine in Pepperland',
+                permalink: true
+              } only %}
+
+              <p>There are some places where Slowpoke is worshiped because of a long-standing belief that whenever Slowpoke yawns, it rains. Carrying food through <code>Fearow’s territory is dangerous</code>. It will snatch the food away from you in a flash! Oddish searches for fertile, nutrient-rich soil, then plants itself. During the daytime, while it is planted, this <code>Pokémon’s feet</code> are thought to change shape and become similar to the roots of trees.</p>
+
+              {% include '@cloudfour/design/typography/demo/code-block.twig' only %}
+
+              <p>In the distant past, they were fairly strong, but they have become gradually weaker over time. Although known for their splendid tail fins, Goldeen apparently compete among themselves to see whose horn is thickest and sharpest. Beautifly has a long mouth like a coiled needle, which is very convenient for collecting pollen from flowers. This Pokémon rides the spring winds as it flits around gathering pollen.</p>
+
+              {% include '@cloudfour/objects/list/list.twig' with {
+                "items": [
+                  "Design Tokens",
+                  "Objects",
+                  "Components",
+                  "Scope",
+                  "Utilities",
+                  "Themes"
+                ],
+                "tag_name": "ul"
+              } only %}
+
+              <p>Its tail is large and covered with a rich, thick fur. The tail becomes increasingly deeper in color as Wartortle ages. The scratches on its shell are evidence of this Pokémon’s toughness as a battler. Lombre’s entire body is covered by a slippery, slimy film.</p>
+            {% endblock %}
+          {% endembed %}
+
+          {% embed '@cloudfour/components/card/card.twig' with {
+            href: '#',
+            class: ' c-card--contained'
+          } only %}
+            {% block content %}
+
+              {% embed '@cloudfour/objects/media/media.twig' with {
+                reverse: true
+              } only %}
+                {% block object %}
+                  {% include '@cloudfour/components/avatar/avatar.twig' with {
+                    src: 'media/team-avatars/aileen.png'
+                  } only %}
+                {% endblock %}
+                {% block content %}
+                  <p><a href="#">Aileen Jeffries</a> Magna ea anim duis fugiat sit labore labore sit nisi eiusmod in consectetur duis id. Aliqua commodo aliqua do fugiat labore consectetur cillum exercitation. <a href="#">Quis do magna</a> and <a href="#">adipisicing</a>.</p>
+                {% endblock %}
+              {% endembed %}
+            {% endblock %}
+          {% endembed %}
```

Comment:
Something I miss from our current design is how this bio info is full-width at small sizes and only becomes constrained at larger ones. The rounded corners are nice, though!